examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
Yeah, I think that makes sense, and it'll simplify the docs a lot. @ncdc  @jrnt30 We do need to make docs changes to `cloud-provider-specifics.md`:

* Update the [basic (no-PVs) example](https://github.com/jrnt30/ark/blob/6483bf0127ab641dcb5765a9cff63d27a9ea77d1/docs/cloud-provider-specifics.md#basic-example-no-pvs) command to include `--snapshot-volumes=false` and `--restore-volumes=false`

* It might be clearer to also update the [snapshot example](https://github.com/jrnt30/ark/blob/6483bf0127ab641dcb5765a9cff63d27a9ea77d1/docs/cloud-provider-specifics.md#snapshot-example-with-pvs) section to either specify `--snapshot-volumes=true` and `--restore-volumes=true`, or remove the flags entirely since its `true` by default

* Remove the first node-labeling example from the snapshot example instructions

Other changes:

* Bump the version (is this major? the behavior seems backwards incompatible even though it's not a huge change...or do we consider it a "bug fix" in which case it's patch?)

* ^ Related to above, consider adding a link in the `CHANGELOG.md` that links back to old docs for the previous version (and a note at the top of `cloud-provider-specifics.md)`? Not sure if this is necessary.